src/main/java/org/elasticsearch/script/ScriptService.java

```diff
@@ -263,10 +264,13 @@ public CompiledScript compileInternal(Script script) {
             return compiled;
         }
 
+        // Name will remain null for inline scripts.
```

Comment:
is this comment correct? seems to me that we assign it only in case of indexed scripts?